src/common/identity/selectors.js

```diff
@@ -387,6 +475,16 @@ export const selectCorporateLegalEntityTypes = createSelector(
 	idType => (idType ? idType.content.enum : [])
 );
 
+// Countries
+
+export const selectCountries = createSelector(
+	state => selectIdAttributeTypeByUrl(state, { attributeTypeUrl: COUNTRY_ATTRIBUTE }),
+	idType => {
+		const { enum: codes, enumNames: names } = idType.content.properties.country;
+		return codes.map((country, index) => ({ country, denonym: names[index] }));
```

Comment:
denonym is incorrect, should be "name"

denonym:
"The word used to identify citizens of the country"

For example: 
Country: "Germany"
Denonym: "German"

Reply:
Fixed on f5f6884